pkg/tsdb/v3iotsdb_integration_test.go

```diff
@@ -357,3 +523,12 @@ func TestDeleteTSDB(t *testing.T) {
 		t.Fatal("Did not delete TSDB properly")
 	}
 }
+
+func iteratorToSlice(it chunkenc.Iterator) []tsdbtest.DataPoint {
+	var result []tsdbtest.DataPoint
+	for it.Next() {
+		t, v := it.At()
```

Comment:
Shouldn't you be checking `Err()`? `chunkenc.Iterator` doesn't really define a contract for it, but it's there.